index.html

```diff
@@ -19,40 +19,186 @@
+               <div class="tools"> <p>html</p>
+                <p>css</p>
+                <p>javaScript</p>
+              </div>
+              <div class="view-project">
+              <button type="button">See Project</button>
+            </div>
+              </li>
+              <li class="work work4">
+                <img class="img1" src="portfolioimgs/Snap4.png" alt="work1Snapshot">
+                  <h1 class="multi">Multi-Post
+                    Stories</h1>
+                  <div class="canopy">
+                  <h3>CANOPY</h3>
+                  <ul class="cards">
+                    <li class="info">Back End Dev</li>
+                    <li class="info">2015</li></ul>
+                  </div>
+                  <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <div class="tools"> <p>html</p>
+                  <p>css</p>
+                  <p>javaScript</p>
+                </div>
+                <div class="view-project">
+                <button type="button">See Project</button>
+              </div>
+                </li>
+         </ul>
+       </section>
+
+       <section class="about-section">
+        <div class="about">
+         <h1 class="aboutMySelf" >About Myself</h1>
+          <p class="text text2 ">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<h3 class="contact2">LET'S CONNECT</h3>
+<ul class="box2">
+<li class="socialMedia">
+    <a class="get-my-contact" href="https://twitter.com/LoukmaneOustani">
+      <img src="portfolioimgs/1.png" alt="twitter">
+     </a>
+ </li>
+ <li class="socialMedia">
+   <a  class="get-my-contact" href="https://www.linkedin.com/in/loukmane-oustani-221668211/">
+     <img src="portfolioimgs/2.png" alt="Linkedin">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a class="get-my-contact" href="#">
+     <img src="portfolioimgs/3.png" alt="M">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a class="get-my-contact" href="https://github.com/L2oukmane">
+     <img src="portfolioimgs/4.png" alt="github">
+   </a>
+ </li>
+ <li class="socialMedia">
+   <a class="get-my-contact" href="#">
+     <img src="portfolioimgs/5.png" alt="peace">
+   </a>
+ </li>
+</ul>
+        </div>
+<div class="resume">
+  <button type="button">Get my resume</button>
+  <ul class="languages">
+ <li>Languages</li>
+ <li><img class="scrollDown" src="portfolioimgs/down.png" alt="down">
+</li>
+  </ul>
+
+  <ul class="html-css">
+    <li class="html-container">
+   <img  class ="html" src="portfolioimgs/HTML.png" alt="html"/> <span class="html-li">HTML</span>
+    </li>
+  <li class="css-container">
+   <img class="css" src="portfolioimgs/CSS.png" alt="css"/> <span class="css-li">CSS</span>
+  </li>
+  </ul>
```

Comment:
[OPTIONAL]: I will suggest you add the third language to your project as in the design.